apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -56,7 +56,8 @@ import { RemoveAllMessages } from './usecases/remove-messages/remove-all-message
 import { RemoveAllMessagesDto } from './dtos/remove-all-messages.dto';
 
 @Controller('/widgets')
-@ApiExcludeController()
+// @ApiExcludeController()
+@ApiTags('Widgets')
```

Comment:
It shouldn't be externally accessible  - why do you need that? :)

Reply:
Sorry,
It was mistake :)
Fixed ✅